karspexet/ticket/models.py

```diff
@@ -1,16 +1,45 @@
 from django.conf import settings
+from django.contrib.postgres.fields import HStoreField
 from django.db import models
+from django.db.models import Q
+from django.utils import timezone
+
+from karspexet.show.models import Show
+from karspexet.venue.models import Seat
 
 
 # TODO: These should probably all be timestamped
 
 
+class ActiveReservationsManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(Q(session_timeout__gt=timezone.now()) | Q(finalized=True))
+
+
 class Reservation(models.Model):
+    show = models.ForeignKey(Show, null=False)
     total = models.IntegerField()
+    tickets = HStoreField()
+    session_timeout = models.DateTimeField()
+    finalized = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    last_modified_at = models.DateTimeField(auto_now=True)
+
+    objects = models.Manager()
+    active = ActiveReservationsManager()
+
+
+class Account(models.Model):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
 
 
 class Ticket(models.Model):
     price = models.IntegerField()
+    ticket_type = models.CharField(max_length=10) # TODO: Validate that ticket type is in ["normal", "student"]
```

Comment:
I think this can be done using [choices](https://docs.djangoproject.com/en/1.11/ref/models/fields/#field-choices
).